lib/ui/semantics.dart

```diff
@@ -248,7 +265,13 @@ class SemanticsFlag {
 
   /// The semantics node has the quality of either being "checked" or "unchecked".
   ///
+  /// This flag is mutually exclusive with [hasToggledState].
```

Comment:
We should enforce this with an assert on the framework side.

Reply:
I was planning to do this in the same place with handle the scopesRoute assert